modules/explorer-core/src/test/scala/org/ergoplatform/explorer/db/repositories/AssetRepoSpec.scala

```diff
@@ -33,8 +34,33 @@ class AssetRepoSpec
     }
   }
 
+  property("insert/getIssuingBoxes") {
+    withLiveRepos[ConnectionIO] { (assetRepo, outputRepo, inputRepo)=>
+      assetRepo.getAllIssuingBoxes.runWithIO() shouldBe empty
+
+      val out = outputGen(true).sample.get
+      val asset = assetGen.sample.get.copy(boxId = out.boxId)
+      import io.estatico.newtype.ops._
+      val inputBoxId = asset.tokenId.toString.coerce[BoxId]
+      val input = inputGen(true).sample.get.copy(txId = out.txId, boxId = inputBoxId)
+      inputRepo.insert(input).runWithIO()
+      assetRepo.insert(asset).runWithIO()
+      outputRepo.insert(out).runWithIO()
+
+      assetRepo.getAllIssuingBoxes.runWithIO() should
+        contain theSameElementsAs List(ExtendedOutput(out, None))
+    }
+  }
+
   private def withLiveRepo[D[_]: LiftConnectionIO: Sync](
     body: AssetRepo[D, fs2.Stream] => Any
   ): Any =
     body(db.repositories.AssetRepo[D])
+
+  private def withLiveRepos[D[_]: LiftConnectionIO: Sync](body:
+                                                          (AssetRepo[D, fs2.Stream],
+                                                            OutputRepo[D, fs2.Stream],
+                                                            InputRepo[D]) => Any): Any =
```

Comment:
Please use scala-fmt to format code according to project configuration

Reply:
Thanks, fixed. Although I enabled scalafmt in IDEA, it still uses its own formatter for line range by default. 🤦‍♂